app/models/node.rb

```diff
@@ -130,7 +130,9 @@ def setup
   def totalviews
     # disabled as impressionist is not currently updating counter_cache; see above
     # self.views + self.legacy_views
-    impressionist_count(filter: :ip_address) + legacy_views
+    cache("totalviews-#{self.id}", expires_in: 60.minutes, skip_digest: true) do
```

Comment:
Maybe `Rails.cache` should be used instead.

Reply:
aha! Let me try.